### PR TITLE
feat: added cart page

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
image

### CODE REVIEW

File: cart-page/cart.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                    <img src="/assets/pngegg.1.png" class="img-fluid" alt="" width="170" height="170">
+                </div>
+
+                <div class="card--body">
+                    <div class="card--header">
+                        <div class="divider--horizontal"></div>
+                        <div class="card--body">
+                            <div class="card--details">
+                                <h4 class="card--title">Areca Plant</h4>
+                                <div class="card--description">Green Arcea Palm Air Purifier Natural Live Plant</div>
```
Comment (Sahil25061999) [inline cart-page/cart.html:83]: can use p tag instead

File: cart-page/cart.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+
+            <div class="card card--horizontal">
+
+                <div class="card--image">
+                    <img src="/assets/pngegplant.png" class="img-fluid" alt="" width="170" height="170">
```
Comment (Sahil25061999) [inline cart-page/cart.html:115]: avoid inline styling

File: cart-page/cart.html [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+                <div class="card--body">
+                    <div class="card--header">
+                        <div class="divider--horizontal"></div>
+                        <div class="card--body">
+                            <div class="card--details">
+                                <h4 class="card--title">Areca Plant</h4>
+                                <div class="card--description">Green Arcea Palm Air Purifier Natural Live Plant</div>
+                                <div class="badge-rating">
+                                    4.5 <i class="fa fa-star" style="font-size:0.9rem; color: white;"></i>
```
Comment (Sahil25061999) [inline cart-page/cart.html:126]: avoid inline styling here too

**Review by Sahil25061999 — APPROVED** [review]
looks good can be merged with minor changes

**Review by mehulsatardekar — APPROVED** [review]
just to add what @Sahil25061999  has suggested is right, avoid inline styling. other than this code is nicely structured and good to merge

**Pull request merged**